components/layout/fragment.rs

```diff
@@ -1051,7 +1051,8 @@ impl Fragment {
                     // Note: We can not precompute the ratio and store it as a float, because
                     // doing so may result one pixel difference in calculation for certain
                     // images, thus make some tests fail.
-                    inline_size * intrinsic_block_size.0 / intrinsic_inline_size.0
```

Comment:
The divide will produce an integer and discard the decimal part, may result in pixel shift.